Functions for diagnosing specific humidity & diagnosing at countries

hdrake opened the pull request.

hdrake committed:
Functions for diagnosing specific humidity & diagnosing at countries

glwagner commented on notebooks/utils.py:
```diff
@@ -21,7 +21,27 @@ def load_jhu_df():
 
 def growth_period(df, region, period=14, threshold=100.):
     if ~((df.loc[region]['Confirmed'] >= threshold).any()):
```
Perhaps write this as 

```python
(df.loc[region]['Confirmed'] < threshold).any():
```

?

hdrake committed:
Plotting specific humidity against growth rates

I diagnosed specific humidity at the grid point nearest the mean latitude and longitude of the data from each country/region. Growth rates were calculated in the 7-14 days following the 100th case in that country, by fitting the confirmed cases with an exponential.

glwagner commented:
Nice work. Might be nice to write out the formulas in notebook 1 to be specific, eg we are calculating `I(t)` during a specific growth interval `[t, t + Δ]`, using the model

```
I = exp(r*I)
```

and using least square to find `r`?
